drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1227,7 +1275,7 @@ function DragAndDropBlock(runtime, element, configuration) {
     };
 
     var focusFirstDraggable = function() {
-        $root.find('.item-bank .option').first().focus();
+        $root.find('.item-bank .option[draggable=true]').first().focus();
```

Comment:
I'm surprised this change was necessary?  Aren't all `.item-bank .option` divs draggable?  (I could be wrong.)

Reply:
With new requirement every placed item will have a placehoder in item bank.

Reply:
https://edx-wiki.atlassian.net/browse/MCKIN-7633